vibegit/cli.py

```diff
@@ -784,16 +791,17 @@ def get_repo() -> git.Repo:
 
 
 def run_commit(debug: bool = False):
-    # For now, only the 'commit' subcommand is implemented directly.
-    # Later, this could use argparse or Typer/Click to handle subcommands.
-    # Example: if args.subcommand == 'commit': await run_commit_workflow()
-
+    """Run the commit workflow with optional debug output.
+    
+    Args:
+        debug: If True, enables additional debug output
+    """
```

Comment:
```suggestion
    """
    # For now, only the 'commit' subcommand is implemented directly.
    # Later, this could use argparse or Typer/Click to handle subcommands.
    # Example: if args.subcommand == 'commit': await run_commit_workflow()
```